### PR TITLE
Fixed the debug menu prefab

### PR DESCRIPTION
**Pull request by kkjamie**
The prefab had some junk in there from testing. It's now been removed

### CODE REVIEW

**Review by StevenNMorrison — APPROVED** [review]
All of these look reasonable, so long as you've tested it.

**Comment by kkjamie** [comment]
I had to make a few extra changes to fix the navigation focus issues

**Review by borntocompile — APPROVED** [review]

File: Scripts/NavigationFocus.cs [hunk 1]
```diff
@@ -16,6 +17,8 @@ public class NavigationFocus : MonoBehaviour
 		[SerializeField]
 		private bool focusOnEnable = true;
 
+		private int framesUntilFocus;
```
Comment (FancySensei) [inline Scripts/NavigationFocus.cs]: Unused var.

File: Scripts/NavigationFocus.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using UnityEngine;
+﻿using System.Collections;
```
Comment (FancySensei) [inline Scripts/NavigationFocus.cs:1]: Unused import.

Comment (kkjamie) [reply Scripts/NavigationFocus.cs:1]: Used for IEnumerator

File: Scripts/NavigationFocus.cs (hunk 1 above)
Comment (FancySensei) [inline Scripts/NavigationFocus.cs]: Unused var.

**Review by FancySensei — APPROVED** [review]

**Pull request merged**